package.json

```diff
@@ -40,8 +40,7 @@
   "activationEvents": [
     "onCommand:extension.fabric8AnalyticsWidgetFullStack",
     "workspaceContains:**/package.json",
-    "workspaceContains:**/pom.xml",
-    "workspaceContains:**/requirements.txt"
```

Comment:
why are you removing `requirements.txt`

Reply:
As python support is not needed in next release.